docs/README.md

```diff
@@ -123,3 +130,420 @@ something like
+      title: "Build a blog in 10 minutes with Booster"
+      content: "I am so excited to write my first post"
+      author: "Boosted developer"
+    }
+  )
+}
+```
+
+```graphql
+mutation {
+  CreatePost(
+    input: {
+      postId: "05670e55-fd31-490e-b585-3a0096db0412"
+      title: "Booster framework rocks"
+      content: "I am so excited for writting the second post"
+      author: "Another boosted developer"
+    }
+  )
+}
+```
+
+The expected response for each of those requests should be:
+
+```json
+{
+  "data": {
+    "CreatePost": true
+  }
+}
+```
+
+Note:
+
+> The IDs are generated on the client-side. For this example, we are not validating ID
+> uniqueness, consider using [a UUID generator](https://www.uuidgenerator.net/version4)
+> when creating more Posts.
+
+##### 7.2 Retrieving all posts
+
+Let's perform a GraphQL `query` that will be hitting our `PostReadModel`:
+
+```graphql
+query {
+  PostReadModels {
+    id
+    title
+    author
+  }
+}
+```
+
+It should response something like:
+
+```json
+{
+  "data": {
+    "PostReadModels": [
+      {
+        "id": "05670e55-fd31-490e-b585-3a0096db0412",
+        "title": "Booster framework rocks",
+        "author": "Another boosted developer"
+      },
+      {
+        "id": "95ddb544-4a60-439f-a0e4-c57e806f2f6e",
+        "title": "Build a blog in 10 minutes with Booster",
+        "author": "Boosted developer"
+      }
+    ]
+  }
+}
+```
+
+##### 7.3 Retrieving specific post
+
+It is also possible to retrieve specific a `Post` by adding the `id` as input, e.g.:
+
+```graphql
+query {
+  PostReadModel(id: "95ddb544-4a60-439f-a0e4-c57e806f2f6e") {
+    id
+    title
+    author
+  }
+}
+```
+
+You should get a response similar to this:
+
+```json
+{
+  "data": {
+    "PostReadModel": {
+      "id": "95ddb544-4a60-439f-a0e4-c57e806f2f6e",
+      "title": "Build a blog in 10 minutes with Booster",
+      "author": "Boosted developer"
+    }
+  }
+}
+```
+
+#### 8. Removing stack
+
+Now, let's destroy all the infrastructure created for running the application in the cloud.
+Execute the following command from the root of the project. For security reasons, you have
+to confirm this action by writing the project's name, in our case `boosted-blog` that is
+the same used when we run `new:project` CLI command.
+
+```bash
+> boost nuke -e production
+
+? Please, enter the app name to confirm deletion of all resources: boosted-blog
+```
+
+Note:
+> Congratulations! You've built a serverless backend in less than 10 minutes. We hope you
+> have enjoyed discovering the magic of the Booster Framework.
+
+#### 9. More functionalities
+
+The are many other options for your serverless backend built with Booster Framework:
+
+- Authorize commands and read models based on different roles
+- Use GraphQL subscriptions
+- Make events to trigger other events
+- Serving static content
+- Reading entities within command handlers to apply domain-driven decisions
+- and much more...
```

Comment:
```suggestion
- Build more complex authorization schemas for commands and read models based on user roles
- Use GraphQL subscriptions to get updates in real-time
- Make events trigger event handlers that can run code or trigger other events
- Deploy static content
- and much more...
```

We're still discussing this => Reading entities within command handlers to apply domain-driven decisions

Reply:
I'd rewrite this part in your suggestion

```diff
- - Make events trigger event handlers that can run code or trigger other events
+ - Make events trigger code or other events
```

Reply:
I finally wrote: `- Make events trigger other events`